pkg/antctl/command_message.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package antctl
+
+import (
+	"fmt"
+	"k8s.io/client-go/rest"
+	"net/http"
+)
+
+func generateMessage(opt *requestOption, result rest.Result) error {
+	var code int
+	_ = result.StatusCode(&code)
+	if err := generate(opt.commandDefinition, opt.args, code); err != nil {
+		return err
+	}
+	return result.Error()
+}
+
+func generate(cd *commandDefinition, args map[string]string, code int) error {
+	switch code {
+	case http.StatusNotFound:
+		return fmt.Errorf("NotFound: %s \"%s\" not found", cd.use, args["name"])
+	}
```

Comment:
what about other error codes?

Reply:
I added case for `http.StatusInternalServerError` and the rest will get `Unknown error`. 

You mentioned this case last time which returns `http.StatusBadRequest`:
```
# antctl get pod-interface -A my-pod
Error: error when requesting https://127.0.0.1:10443/podinterface?all-namespaces=true&name=my-pod&namespace=default: the server rejected our request for an unknown reason
```
But we don't have `-A` flag now so it is not considered.
These error codes are so far what I found in handlers and we can add more in the future.

Reply:
I saw that you added a default case with `return fmt.Errorf("Unknown error")`, that's what I was looking for